configuration-service/handlers/project.go

```diff
@@ -160,8 +156,6 @@ func PutProjectProjectNameHandlerFunc(params project.PutProjectProjectNameParams
 
 // DeleteProjectProjectNameHandlerFunc deletes a project
 func DeleteProjectProjectNameHandlerFunc(params project.DeleteProjectProjectNameParams) middleware.Responder {
-	common.Lock()
```

Comment:
Please add the lock here again, as deleting should not work while a lock is active.